lib/configParser.js

```diff
@@ -95,7 +95,7 @@ ConfigParser.resolveFilePatterns =
   if (patterns) {
     for (var i = 0; i < patterns.length; ++i) {
       var fileName = patterns[i];
-      var matches = glob.sync(fileName, {cwd: cwd});
+      var matches = glob.hasMagic(fileName) ? glob.sync(fileName, {cwd: cwd}) : [fileName];
```

Comment:
does it take into account of multiple feature line numbers which is in cucumber 0.9.2 like abc.feature:4:9:11